### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by Hofer-Julian** [bilelmoussaoui/flatpak-vscode#29]
We should test in order to avoid regressions.

Unit tests:
- [ ] isFlatpak 
- [ ] parseManifest 
- [ ] findManifests 

GUI tests:
- [ ] compile&run a simple apps with all supported buildsystems

**Comment by SeaDve** [comment]
I'm not sure if testing isFlatpak/isSandboxed is possible

**Comment by bilelmoussaoui** [comment]
Indeed, I think the most important ones to test are things like the generated build commands for each build system.
